net/zerotier/src/opnsense/service/templates/OPNsense/zerotier/zerotier

```diff
@@ -6,3 +6,4 @@ zerotier_enable="YES"
 {% else %}
 zerotier_enable="NO"
 {% endif %}
+zerotier_var_mfs="/var/db/zerotier-one"
```

Comment:
into enabled block please :)

Reply:
Hi, as discussed elsewhere, it's okay to live outside for the moment, since the user may disable the service (thus creating a zerotier conf file with just zerotier_enabled="NO" in it, reboot the router, then re-enable zerotier and be surprised that their information has disappeared.

I'm looking at an even longer term solution where I can tell zerotier where to store its db files.

For now, I suppose this is a reasonable stop-gap measure.

-=david=-